README.md

```diff
@@ -26,6 +25,7 @@ Contents
   - [Faker.Name](doc/name.md)
   - [Faker.Number](doc/number.md)
   - [Faker.Pokemon](doc/pokemon.md)
+  - [Faker.RickAndMorty](doc/rick_and_morty.md)
```

Comment:
Shuffle this down one place

Reply:
This and the rest of the requested changes should be done now so I'll resolve them all.